### ISSUE TITLE
schedule cannot be saved when the 'skip' field is zero

### ISSUE THREAD

**Issue by joergmschulz** [mziech/nextcloud-calendar-news#9]
Weekly Schedule saving failed for me as long as I left the 'skip' field empty. After setting it to '0' I could save my schedule.
Suggestion: pre-define value '0' for the skip field.

**Comment by mziech** [comment]
I agree, but I also think the whole form has a lot more validation issues ;)

**Comment by mziech** [comment]
Fixed in 1.1.5

**Comment by MarkWieczorek** [comment]
Unfortunately, I am having the same problem with version 1.1.5. Leaving the skip field empty fails to save the configuration.

**Comment by mziech** [comment]
Hmm, yes, the validation issue still exists, the form is merely prefilled with 0 ... I guess the validation needs to be fixed for good.
